templates/search/blocks/search_results_tag_frequency.html

```diff
@@ -5,41 +5,46 @@
         <p class="tag-frequency__title">Filter by:</p>
         <ul class="tag-frequency__tags-list">
             <li>
-                <a href="?{% query_string_include 'chart_data_type' 'LOC' %}" class="ohos-tag ohos-tag--location">
+<!--                <a href="?{% query_string_include 'chart_data_type' 'LOC' %}" class="ohos-tag ohos-tag&#45;&#45;location">-->
+                    <a href="/search/catalogue/?group=community&vis_view=tag&chart_data_type=LOC" class="ohos-tag ohos-tag--location">
```

Comment:
1. The comment has introduced `&#45;&#45` which is not part of the original code. For Non JS code we can use the `{% comment %}` template tag like this :  `{% comment %} <a href="?{% query_string_include 'chart_data_type' 'LOC' %}" class="ohos-tag ohos-tag--location"> {% endcomment %}`

Why is the link commented out? 

2.  Why is the link changed from the original one?

Reply:
The commented code has now been removed.

The link has changed to reflect the behaviour required in the ticket, so that selected items are not persitent across the menu items : Location, Person, Organisation, Miscellaneous, Date.

Reply:
Ok thanks, I've related this to - _Deselect tags selected when changing tag type view or visualisation or bucket_